### ISSUE TITLE
FCN Acronym on Cryptopia / HitBTC

### ISSUE THREAD

**Issue by PhinixPhire** [ccxt-dev/ccxt#357]
Good morning,

Cryptopia uses FCN for "FacileCoin"
HitBTC uses FCN for "FantomCoin"

FacileCoin is the lesser coin between the two, I think. FacileCoin doesn't even appear in the CoinMarketCap at all. Suggest we make another acronym for FacileCoin in cryptopia's so as to not cross over to FantomCoin. 

How would you like to handle this?

**Comment by kroitor** [comment]
Hi! Thx again for your involvement, I added a mapping for it and it should work as of next version 1.9.211 in approx 5 minutes. Cheers!

**Comment by PhinixPhire** [comment]
FCN is already FacileCoin on cryptopia. Sorry if I was unclear. My concern is that it can be mistaken as FantomCoin, as CoinMarketCap and all exchanges I've seen use FCN for FantomCoin...

Should we change FacileCoin to FACN, maybe? 

Cryptopia has so very many coins. I'm sure there are others I'll come across that need adjustment. But I figure I should probably report, suggest and discuss before doing a pull request, yes?

**Comment by kroitor** [comment]
@PhinixPhire let me explain, to remove the ambiguity of FCN there is literally only one way, and you correctly determined the prevailing FantomCoin. So together we decide that FCN should stand for FantomCoin as it does for most of exchanges. Whereas on Cryptopia it should be renamed to something very clear and explicit. The first thing that comes to my mind is to use full name of the coin, so we declare that FCN on Cryptopia was inconsistent with the rest of the world, and we are renaming it to its full name.

FCN on Cryptopia, BEFORE:
```
3526 | FCN/BTC  | FCN | BTC   | [object Object] | 0.002 | 0.002 | [object Object] | [object Object]
3529 | FCN/DOGE | FCN | DOGE  | [object Object] | 0.002 | 0.002 | [object Object] | [object Object]
3528 | FCN/LTC  | FCN | LTC   | [object Object] | 0.002 | 0.002 | [object Object] | [object Object]
```

Facilecoin on Cryptopia, AFTER:
```
3526 | Facilecoin/BTC  | Facilecoin | BTC   | [object Object] | 0.002 | 0.002 | [object Object] | [object Object]
3529 | Facilecoin/DOGE | Facilecoin | DOGE  | [object Object] | 0.002 | 0.002 | [object Object] | [object Object]
3528 | Facilecoin/LTC  | Facilecoin | LTC   | [object Object] | 0.002 | 0.002 | [object Object] | [object Object]
```

So, as you see the symbol itself has changed on Cryptopia to resolve the conflict of two coins designated obviously incorrectly and bring more consistency to the cross-exchange naming.

In other words, on Cryptopia there is no FCN/BTC pair anymore, but there is now Facilecoin/BTC, and that is the symbol string that users should be passing to methods of Cryptopia for that pair ;)

**Comment by PhinixPhire** [comment]
Perfect! 
You're right, I was misinterpreting. Thanks so much!

**Comment by kroitor** [comment]
> But I figure I should probably report, suggest and discuss before doing a pull request, yes?

Your PRs are welcome! I highly recommend reading the [CONTRIBUTING](https://github.com/ccxt-dev/ccxt/blob/master/CONTRIBUTING.md) guidelines first ;)
